mkosi/distributions/opensuse.py

```diff
@@ -21,47 +23,128 @@ def filesystem(cls) -> str:
 
     @classmethod
     def install(cls, state: "MkosiState") -> None:
```

Comment:
```suggestion
    def install(cls, state: MkosiState) -> None:
```
This is actually a copy paste error introduced earlier, so the string quoting `"MkosiState"` is unnecessary in the rest.